### ISSUE TITLE
mdJson metadata object required

### ISSUE THREAD

**Issue by stansmith907** [adiwg/mdJson-schemas#188]
Maybe.  Currently at the top schema level only the schema and contact objects are required.  It seems the metadata object should also be required - at least if the record is being sent through the translator.  I can see saving an mdJson object with just contacts for the mdEditor, but that should never be passed to the mdTranslator.  From an mdTranslator perspective there should always be a metadata object, thus the schema should back that up.

**Comment by jlblcc** [comment]
I can implement this now, but we did discuss the possibility of just having a dataDictionary, i.e. for ISO 19110 output. Do we want to revisit that or just require metadata only?

**Comment by stansmith907** [comment]
Yep, you're right forgot about dictionary only option.  Close this one.

**Comment by jlblcc** [comment]
We can require at least one of dataDictionary or metadata?

**Comment by stansmith907** [comment]
Unless we also want to just hold a bundle of contacts.  Although that may not be of any value to the translator, the editor may also want to use the schema.  We can just leave it as is for the schema and this can be a translator check only.
